src/main/java/seedu/address/storage/JsonAdaptedRoom.java

```diff
@@ -4,24 +4,33 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.patient.Patient;
 import seedu.address.model.room.Room;
 
 public class JsonAdaptedRoom {
```

Comment:
would be good to have a class header for java docs!

Reply:
Okay will add, thanks for pointing out.